tensilelite/requirements.txt

```diff
@@ -1,7 +1,7 @@
 dataclasses; python_version == '3.6'
 pyyaml
 msgpack
-joblib
+joblib>=1.4.0
```

Comment:
SLES has joblib 1.1.1.

Reply:
Hmm thanks for the reminding! originally I implemented with `concurrent` module, and then switched to `joblib.Parallel` for simplicity, maybe we need such fallback version if `joblib` >= 1.4 not available.